.github/workflows/integration.yml

```diff
@@ -0,0 +1,23 @@
+name: Integration
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 0 * * *'  # daily
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
In the interest of faster test runs and reduced complexity, this doesn't run on multiple platforms or Python versions. I'm assuming that failures related to those things would be uncovered by the other tests.